client/src/index.js

```diff
@@ -28,51 +40,90 @@ function persistReducer(state = { }, action) {
   }
 }
 
+function commentsReducer(state = {}, action) {
+  switch (action.type) {
+    case LOAD_ALL:
+      return action.comments
+    /*
+    case RELOAD_COMMENT:
+      return action
+
+    case PREVIEW_COMMENT:
+      return action
+    */
+    default:
+      return state
+  }
+}
+
 const reducer = combineReducers({
   persist: persistReducer,
+  comments: commentsReducer,
 })
 
 const store = createStore(reducer)
```

Comment:
Could everything above this be in separate file?

Reply:
Yea reducers should be in some files probably, so should the stoe

Reply:
Ok, let's leave it for later